src/client/src/app/modules/shared/services/resource/resource.service.ts

```diff
@@ -81,8 +81,8 @@ export class ResourceService {
       };
       this.get(option).subscribe(
         (data: ServerResponse) => {
-          this.messages = _.merge({},  data.result.creation.messages, data.result.consumption.messages);
-          this.frmelmnts = _.merge({}, data.result.creation.frmelmnts, data.result.consumption.frmelmnts);
+          this.messages = _.merge({},  data.result.consumption.messages);
```

Comment:
change to 
this.messages = data.result.consumption.messages;
this.frmelmnts = data.result.consumption.frmelmnts;